test/widgets/subscription_list_test.dart

```diff
@@ -190,4 +202,43 @@ void main() {
     check(tester.widget<UnreadCountBadge>(find.byType(UnreadCountBadge)).backgroundColor)
       .equals(swatch);
   });
+
+  testWidgets('muted streams are displayed as faded', (tester) async {
+    final stream1 = eg.stream(name: 'Stream 1');
+    final stream2 = eg.stream(streamId: 2, name: 'Stream 2');
```

Comment:
Why pass `streamId` when making `stream2` but not `stream1`? Does this test need a specific stream ID for `stream2` but not `stream1`?

…Ah, I think I see: this test doesn't particularly care about the specific stream-ID values, but it does need them to be different from each other. That's accomplished by letting `stream1` be the default value 123, and giving `stream2` some different value.

Slightly nicer would be to just pass `streamId: 1` and `streamId: 2`. That would remove this surprising asymmetry that I noticed.

Better than that, though, would be to make a prep commit that takes care of this TODO in `eg.stream`:

```dart
    streamId: streamId ?? 123, // TODO generate example IDs
```

and then not pass `streamId` when making `stream1` or `stream2`, because they'll be automatically generated and different from each other. Like we did for `eg.user` in bb78e6ce, and `eg.streamMessage`/`eg.dmMessage` in 13e54f60b.